pkg/scanners/trivy/types.go

```diff
@@ -62,7 +62,7 @@ func (s *ImageScanner) Scan(img eraserv1alpha1.Image) (ScanStatus, error) {
 		ref := refs[i]
 		log.Info("scanning image with ref", "ref", ref)
 
-		dockerImage, cleanup, err := fanalImage.NewContainerImage(s.ctx, ref, s.scanConfig.dockerOptions, s.imageSourceOptions...)
+		dockerImage, cleanup, err := fanalImage.NewContainerImage(context.Background(), ref, s.scanConfig.dockerOptions, s.imageSourceOptions...)
```

Comment:
This may be the right place to put a per-image timeout, but I'm not certain.

Reply:
I put the per-image scanner timeout as part of the context in the ImageScanner struct, since it uses that context in each Scan() call for each image

Reply:
nevermind, I think I understand. I created a new context for each scan instead with the specified timeout to make it per-image